File: demo/springboot/demo/src/main/java/com/demo/domain/dto/http/resonse/OssFileContentUpdateResponse.java
package com.demo.domain.dto.http.resonse;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

@Data
public class OssFileContentUpdateResponse {
    @JsonFormat(shape = JsonFormat.Shape.STRING)
    private Long fileId;
    private String originFileName;
    private String url;
    private Boolean updated;
    private String message;
}

File: demo/kmp/shared/src/commonMain/kotlin/com/vectordemo/domain/dto/http/response/OssFileContentUpdateResponse.kt
package com.vectordemo.domain.dto.http.response

import kotlinx.serialization.Serializable

@Serializable
data class OssFileContentUpdateResponse(
    val fileId: String? = null,
    val originFileName: String? = null,
    val url: String? = null,
    val updated: Boolean? = null,
    val message: String? = null,
)
